OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/http/impl/OptionalHeaderValues.kt

```diff
@@ -0,0 +1,5 @@
+package com.onesignal.core.internal.http.impl
+
+data class OptionalHeaderValues(
+    var cacheKey: String? = null,
```

Comment:
It doesn't look like we ever assign this value (or would ever want too). Can we make this immutable by changing this `var` into `val`? You can then use the constructor to set the value in one of two ways:
1. ` OptionalHeaderValues("valueOfCacheKey")`
2. ` OptionalHeaderValues(cacheKey = "valueOfCacheKey")`